CONTRIBUTING.md

```diff
@@ -61,6 +62,20 @@ For large fixes, please build and test the documentation before submitting the P
 haven't accidentally introduced any layout or formatting issues. You should also make sure that your
 commit message follows the **[Commit Message Guidelines][developers.commits]**.
 
+### <a name="updating"></a> Updating Documentation through Docusaurus
+
+Docusaurus is used for maintaining our documentation. The process for this involves generation of
+markdown files concerning the API, as well as incrementing the docs version.
+
+<GENERATION OF `cicero-api.md` and `ergo-api.md`>
+Converts JSDoc to Markdown
+
+<PR APPROVAL VERRSION INCREMENT>
+
+  ```text
+    npm run version X.X.X
```

Comment:
Should point to Docusaurus documentation, and/or indicate to be in the `./website` directory.

Reply:
This is still waiting on me understanding the whole process so that it isn't this confusing in the future.